#inputs
# Gripper control has two control methods: force_control and position_control
# force_control: closes the gripper using force control: 0 super weak!, 100 maximum force (can damage stuff)
# position_control: closes the gripper to a certain width in millimeters (0 totally close, 110 fully open), use it before grasping an object (i.e., the space is too narrow)
bool GripperClose                 # choose between force_control(True) and position_control(False)
bool SuctionOn                    # suction ON/OFF
bool lin_act_gripper_extended
bool lin_act_suction_extended
std_msgs/Int16 gripper_control_parameter  # set the force (when GripperClose = 1 => force_control) or the width (when GripperClose = 0 => position_control)
std_msgs/Int16 suction_force              # suction force values from 0 (super weak) to 40 (full power)
---
#outputs
bool succeeded
